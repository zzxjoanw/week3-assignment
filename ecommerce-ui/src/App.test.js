import React from 'react';
import renderer from 'react-test-renderer';
import CardHolder from './components/cardHolder/cardHolder';
import Cart from './components/cart/cart';

it('renders an empty cart', () => {
  const sites = [];
  const emptyCart = renderer.create(<Cart sites={sites} />).toJSON();
  expect(emptyCart).toMatchSnapshot();
});

it('renders a cart with items in it', () => {
  const sites = ['Denver'];
  const cart = renderer.create(<Cart sites={sites} />).toJSON();
  expect(cart).toMatchSnapshot();
});