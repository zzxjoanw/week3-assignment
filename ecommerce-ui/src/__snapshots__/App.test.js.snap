// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a cart with items in it 1`] = `
<div
  id="cart"
>
  <div
    id="cartTop"
  >
    <span>
      Hello.
    </span>
    <span
      className="reservation"
    />
  </div>
  <ul>
    <li>
      Denver
      <button>
        Delete
      </button>
    </li>
  </ul>
</div>
`;

exports[`renders an empty cart 1`] = `
<div
  id="cart"
>
  <div
    id="cartTop"
  >
    <span>
      Hello.
    </span>
    <span
      className="reservation"
    />
  </div>
  <ul />
</div>
`;
